### DISCUSSION TITLE
Girl in Grass Animation (WIP)

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#367] [Show and tell]
This is a work in progress of a simple animation gif I am making. So happy I finished animating the grass it was so tedious(I am sick and tired of grass so much work😩😩😩). I hope to finished the animation by Monday.
![IMG_1926](https://github.com/keiwando/toonsquid/assets/88890833/050f6f2d-02d1-4459-85e6-8d783555bbb6)

**Comment by ota-works** [comment]
I'd suggest you have some areas that the grass intersect with the character remain, and remove the other details. Of course keep some of the flowers to blow in the wind.
The other areas that will be removed can be animated like how Ghibli animate their grass by "drawing" gust of wind blowing against the ground.

Just a suggestion, but great work so far!!! Was all of this done in Toonsquid?

**Reply by Lioncub901** [reply]
Thanks for the feedback I will keep it in mind when I animate grass again (Sorry can’t really change it since there is a due date for this animation). 

Yes, it was done in ToonSquid.
